_posts/2018-08-06-Kickstarter-Trajectory-2018-Summer.md

```diff
@@ -0,0 +1,175 @@
+This is very important to Ordina because all its consultants need to always have this information in the back of their minds when working at a customer.
+We also followed a more technical security course which explained some of the most common attacks and how to avoid them.
+
+## Backend
+
+### Java
+In this lecture given by [Yannick De Turck](/author/yannick-de-turck/){:target="_blank"}, we were introduced to all new features of Java 7 and 8 aswell as Java 9.
+
+We started off with Java 7 where we learned:
+* Switch-statement with String values
+* Automatic Resource Management
+* Diamond Syntax
+* Better exception handling with multi-catch
+* Literal Enhancements
+* The new IO API
+* Fork Join Framework
+* JVM Enhancements
+
+Java 8 also introduced some useful new features:
+* Lambda Expressions
+* Extension Methods
+* Functional Interfaces
+* Method and Constructor References
+* Streams and Bulk Data Operations for Collections
+* Removal of PermGen
+* New Date & Time API
+* New Default API for Base64 Encoding
+* Improvements for Annotations
+* Performance Improvements
+
+Last but not least we had an introduction of Java 9:
+* Project Jigsaw: Modules
+* Project Kulla: JShell
+* Factory Methods for collections
+* Diamond operator for anonymous inner classes
+* Try-with-resources enhancement
+* CompletableFuture API improvements
+* Underscore ('_') character is a keyword
+* Private methods in interfaces
+* HTTP 2.0 Client
+* Process API Improvements
+* Reactive Streams
+* Optional improvements
+* Collectors Improvements
+* Stream improvements
+
+In the afternoon we made a few exercises on these new features and improvements which gave us a brighter view on the possibilities within present java development.
+
+### Spring and Spring Boot
+The lectures of Spring and Spring Boot were given by [Ken Coenen](/author/ken-coenen){:target="_blank"}. 
+These were spread over two days where the first day was an overal introduction to Spring fundamentals, followed by a second day where we have put everything into practice.
+
+#### Day 1:
+The first day we got an introduction to Spring and Spring Boot about the core fundamentals and concepts of JPA, Beans and Application contexts. 
+After that we went further into the features of the framework where we were introduced to Spring Web Services and Spring Security.
+
+#### Day 2:
+On the second day we made a small project where we created a backend application to fetch all information of different digital coins (Cryptocurrencies). 
+We learned how to read data from an API using DTO's and store them into a database. At the end of the day we had a fully working backend application which fetched all information automatically and exposed it to different endpoints.
+
+<img alt="SpringBoot" src="{{ '/img/2018-08-06-Kickstarter-Trajectory-2018-Summer/be-course.png' | prepend: site.baseurl }}" class="image fit">
+
+### Microservices
+This course gave a brief overview of the microservices architecture. 
```

Comment:
Also refer to the teacher(s) (idem dito for the other sessions), seeing as you guys referred to them in the previous two sessions in order to be consistent.